base/services/wlansvc/rpcserver.c

```diff
@@ -64,7 +64,7 @@ PWLANSVCHANDLE WlanSvcGetHandleEntry(LPWLANSVC_RPC_HANDLE ClientHandle)
     return NULL;
 }
 
-DWORD _RpcOpenHandle(
+DWORD WINAPI _RpcOpenHandle(
```

Comment:
In my MSVC build's wlansvc_s.h this is explicit cdecl.

```
DWORD __cdecl _RpcOpenHandle(
    WLANSVC_HANDLE szMachineName,
    DWORD dwClientVersion,
    DWORD *pdwNegotiatedVersion,
    WLANSVC_RPC_HANDLE *phClientHandle);
```

What's clang complaining about?

Reply:
wlansvc_s.c(187,5) :  warning: cast between incompatible calling conventions 'cdecl' and 'stdcall'; calls through this pointer may abort at runtime
(SERVER_ROUTINE)_RpcOpenHandle,
^~~~~~~~~~~~~~~~~~~~~~~~

Reply:
Yeah, SERVER_ROUTINE is a generic function pointer type that doesn't necessarily match the actual function. PVOID might have been a better choice.
In any case, I think we need to silence this warning, I think these changes are incorrect. We must trust widl to do the right thing here.

Reply:
Noted.